### PR TITLE
End user docs for apiserver

### PR DESCRIPTION
**Pull request by zugao**
This PR adds end user documentation for our AppCat API Server.

### CODE REVIEW

File: docs/modules/ROOT/pages/apiserver/how-to.adoc [hunk 1]
```diff
@@ -0,0 +1,25 @@
+= AppCat API Server
+
+Our AppCat API Server lets you query the state of AppCat services in the cluster. The operations can be performed through the kubectl command-line interface or you can also access the API directly using REST calls. Below is a list of available operations.
```
Comment (Kidswiss) [inline docs/modules/ROOT/pages/apiserver/how-to.adoc:3]: Either provider an example of a REST call or remove the mention of it.

**Review by Kidswiss — CHANGES_REQUESTED** [review]

File: docs/modules/ROOT/pages/apiserver/how-to.adoc (hunk 1 above)
Comment (zugao) [reply docs/modules/ROOT/pages/apiserver/how-to.adoc]: Do we need a REST call to be documented?

Comment (Kidswiss) [reply docs/modules/ROOT/pages/apiserver/how-to.adoc]: If we need it at any point we can add the docs for it then. But currently I don't think so. Most interaction with the API server will be through a proper k8s client anyway.

Comment (zugao) [reply docs/modules/ROOT/pages/apiserver/how-to.adoc]: I removed the REST reference

**Pull request merged**